templates/store/store.html

```diff
@@ -306,15 +311,17 @@ <h3 class="p-media-object__title">
   <div class="row">
     <div class="col-12">
       <h2>Databases</h2>
-      <div class="p-strip--dark u-clearfix" style="background:#2089B5;padding:1rem">
-        <div class="col-6 u-align--center">
-          <img src="https://assets.ubuntu.com/v1/90e476fc-mysql-promo.png" alt="">
-        </div>
-        <div class="col-6">
-          <h4>MySQL</h4>
-          <p>By <a href="{{ url_for('jaasstore.user_details', username='mysql-charmers') }}" class="p-link--strong">mysql-charmers</a></p>
-          <p>MySQL is a fast, stable and true multi-user, multi-threaded SQL database server. Its main goals are speed, robustness and ease of use.</p>
-          <a href="{{ url_for('jaasstore.details', charm_or_bundle_name='mysql') }}" class="p-button--neutral">View the charm</a>
+      <div class="p-card pictogram-bg">
+        <div class="u-equal-height">
+          <div class="col-6">
+            <h4>MySQL</h4>
+            <p>By <a href="{{ url_for('jaasstore.user_details', username='mysql-charmers') }}" class="p-link--strong">mysql-charmers</a></p>
+            <p>MySQL is a fast, stable and true multi-user, multi-threaded SQL database server. Its main goals are speed, robustness and ease of use.</p>
+            <a href="{{ url_for('jaasstore.details', charm_or_bundle_name='mysql') }}" class="p-button--neutral">View the charm</a>
+          </div>
+          <div class="col-6 u-align--center u-vertically-center u-hide--small prefix-2">
+            <img width="144" src="https://assets.ubuntu.com/v1/80cd67d3-mysql.svg" alt="mySQL logo">
```

Comment:
How do we feel about using html attrs for image width? I feel like this can cause confusion when applying styles css vs html.